diff -up chromium-75.0.3770.80/third_party/webrtc/rtc_base/physical_socket_server.h.SIOCGSTAMP chromium-75.0.3770.80/third_party/webrtc/rtc_base/physical_socket_server.h
--- chromium-75.0.3770.80/third_party/webrtc/rtc_base/physical_socket_server.h.SIOCGSTAMP	2019-06-06 10:04:57.050403639 -0400
+++ chromium-75.0.3770.80/third_party/webrtc/rtc_base/physical_socket_server.h	2019-06-06 10:06:03.975121688 -0400
@@ -16,6 +16,11 @@
 #define WEBRTC_USE_EPOLL 1
 #endif
 
+// for SIOCGSTAMP in Linux 5.2
+#if defined(WEBRTC_LINUX)
+#include <linux/sockios.h>
+#endif
+
 #include <memory>
 #include <set>
 #include <vector>

