diff -urN chromium-62.0.3202.45.orig/content/network/network_service_impl.cc chromium-62.0.3202.45/content/network/network_service_impl.cc
--- chromium-62.0.3202.45.orig/content/network/network_service_impl.cc	2017-10-05 13:28:37.144228686 +0300
+++ chromium-62.0.3202.45/content/network/network_service_impl.cc	2017-10-06 13:42:53.571499640 +0300
@@ -90,7 +90,7 @@
       base::MakeUnique<NetworkContext>(this, std::move(request),
                                        std::move(params), std::move(builder));
   *url_request_context = network_context->url_request_context();
-  return network_context;
+  return std::unique_ptr<content::NetworkContext>(network_context.release());
 }

 std::unique_ptr<NetworkServiceImpl> NetworkServiceImpl::CreateForTesting() {
