diff -up chromium-56.0.2924.87/third_party/WebKit/Source/BUILD.gn.permissive chromium-56.0.2924.87/third_party/WebKit/Source/BUILD.gn
--- chromium-56.0.2924.87/third_party/WebKit/Source/BUILD.gn.permissive	2017-02-13 12:32:23.419665971 -0500
+++ chromium-56.0.2924.87/third_party/WebKit/Source/BUILD.gn	2017-02-13 12:33:16.146629190 -0500
@@ -54,6 +54,9 @@ config("config") {
   cflags = []
   defines = []
 
+  # error: there are no arguments to 'swapAnchor' that depend on a template parameter, so a declaration of 'swapAnchor' must be available [-fpermissive]
+  cflags += [ "-fpermissive" ]
+
   if (is_win) {
     cflags += [
       "/wd4305",  # Truncation from 'type1' to 'type2'.
