diff -up chromium-63.0.3239.84/build/linux/unbundle/freetype.gn.fixunbundle chromium-63.0.3239.84/build/linux/unbundle/freetype.gn
--- chromium-63.0.3239.84/build/linux/unbundle/freetype.gn.fixunbundle	2017-12-07 12:07:48.652161826 -0500
+++ chromium-63.0.3239.84/build/linux/unbundle/freetype.gn	2017-12-07 12:08:52.514919585 -0500
@@ -117,6 +117,14 @@ source_set("freetype") {
   public_configs = [ ":freetype_config" ]
 }
 
+source_set("freetype_source") {
+  visibility = [ "//third_party:freetype_harfbuzz" ]
+  deps = [
+    ":freetype_shim",
+  ]
+  public_configs = [ ":freetype_config" ]
+}
+
 source_set("bootstrap_freetype_for_harfbuzz") {
   deps = [
     ":freetype_shim",
diff -up chromium-63.0.3239.84/build/linux/unbundle/harfbuzz-ng.gn.fixunbundle chromium-63.0.3239.84/build/linux/unbundle/harfbuzz-ng.gn
--- chromium-63.0.3239.84/build/linux/unbundle/harfbuzz-ng.gn.fixunbundle	2017-12-07 12:09:57.057661738 -0500
+++ chromium-63.0.3239.84/build/linux/unbundle/harfbuzz-ng.gn	2017-12-07 12:10:03.225541404 -0500
@@ -24,6 +24,14 @@ group("harfbuzz-ng") {
   ]
 }
 
+source_set("harfbuzz_source") {
+  visibility = [ "//third_party:freetype_harfbuzz" ]
+  deps = [
+    ":harfbuzz_shim",
+  ]
+  public_configs = [ ":system_harfbuzz" ]
+}
+
 source_set("harfbuzz-ng-ft") {
   deps = [
     ":harfbuzz_shim",
