diff -up chromium-60.0.3112.90/third_party/angle/src/vulkan_support/BUILD.gn.c99 chromium-60.0.3112.90/third_party/angle/src/vulkan_support/BUILD.gn
--- chromium-60.0.3112.90/third_party/angle/src/vulkan_support/BUILD.gn.c99	2017-08-14 10:40:48.028799820 -0400
+++ chromium-60.0.3112.90/third_party/angle/src/vulkan_support/BUILD.gn	2017-08-14 10:41:43.054734910 -0400
@@ -180,6 +180,7 @@ config("vulkan_internal_config") {
     cflags += [ "-Wno-ignored-attributes" ]
   }
   if (is_linux) {
+    cflags += [ "-std=c99" ]
     defines += [
       "SYSCONFDIR=\"/etc\"",
       "FALLBACK_CONFIG_DIRS=\"/etc/xdg\"",
