diff -up chromium-63.0.3239.84/remoting/host/linux/BUILD.gn.nolibc++ chromium-63.0.3239.84/remoting/host/linux/BUILD.gn
--- chromium-63.0.3239.84/remoting/host/linux/BUILD.gn.nolibc++	2017-12-07 11:55:43.173273434 -0500
+++ chromium-63.0.3239.84/remoting/host/linux/BUILD.gn	2017-12-07 11:55:53.244077772 -0500
@@ -62,11 +62,9 @@ if (enable_me2me_host) {
     if (is_component_build) {
       sources += [
         "$root_build_dir/libbase.so",
-        "$root_build_dir/libc++.so",
       ]
       deps += [
         "//base:base",
-        "//buildtools/third_party/libc++:libc++",
       ]
     }
   }
