diff -up chromium-60.0.3112.90/third_party/WebKit/Source/core/layout/ng/ng_block_layout_algorithm.cc.tryfix chromium-60.0.3112.90/third_party/WebKit/Source/core/layout/ng/ng_block_layout_algorithm.cc
--- chromium-60.0.3112.90/third_party/WebKit/Source/core/layout/ng/ng_block_layout_algorithm.cc.tryfix	2017-08-09 14:45:17.619172987 -0400
+++ chromium-60.0.3112.90/third_party/WebKit/Source/core/layout/ng/ng_block_layout_algorithm.cc	2017-08-09 14:46:07.375213058 -0400
@@ -382,7 +382,7 @@ NGLogicalOffset NGBlockLayoutAlgorithm::
 
   NGLogicalOffset child_bfc_offset = curr_bfc_offset_;
   child_bfc_offset.inline_offset +=
-      {border_and_padding_.inline_start + curr_child_margins_.inline_start};
+      {{border_and_padding_.inline_start + curr_child_margins_.inline_start}};
 
   // Append the current margin strut with child's block start margin.
   // Non empty border/padding, and new FC use cases are handled inside of the
@@ -396,8 +396,8 @@ NGLogicalOffset NGBlockLayoutAlgorithm::
   // Should collapse margins if our child is a legacy block.
   if (IsLegacyBlock(*child)) {
     curr_bfc_offset_ +=
-        {border_and_padding_.inline_start + curr_child_margins_.inline_start,
-         curr_margin_strut_.Sum()};
+        {{border_and_padding_.inline_start + curr_child_margins_.inline_start,
+         curr_margin_strut_.Sum()}};
     MaybeUpdateFragmentBfcOffset(ConstraintSpace(), curr_bfc_offset_,
                                  &container_builder_);
     PositionPendingFloats(curr_bfc_offset_.block_offset, &container_builder_,
@@ -526,8 +526,8 @@ NGLogicalOffset NGBlockLayoutAlgorithm::
   margin_strut.Append(curr_child_margins_.block_end);
 
   curr_bfc_offset_ +=
-      {border_and_padding_.inline_start + curr_child_margins_.inline_start,
-       margin_strut.Sum()};
+      {{border_and_padding_.inline_start + curr_child_margins_.inline_start,
+       margin_strut.Sum()}};
   AdjustToClearance(space.ClearanceOffset(), &curr_bfc_offset_);
   PositionPendingFloatsFromOffset(
       curr_bfc_offset_.block_offset, curr_bfc_offset_.block_offset,
