diff -urN chromium-56.0.2924.59.orig/third_party/icu/icu.gyp chromium-56.0.2924.59/third_party/icu/icu.gyp
--- chromium-56.0.2924.59.orig/third_party/icu/icu.gyp	2017-01-16 12:13:25.234506480 +0300
+++ chromium-56.0.2924.59/third_party/icu/icu.gyp	2017-01-16 12:14:27.375292768 +0300
@@ -39,7 +39,7 @@
       }],
       ['(OS=="linux" or OS=="freebsd" or OS=="openbsd" or OS=="solaris" \
          or OS=="netbsd" or OS=="mac" or OS=="android" or OS=="qnx") and \
-        (target_arch=="arm" or target_arch=="ia32" or \
+        (target_arch=="ia32" or \
          target_arch=="mipsel" or target_arch=="mips" or \
          target_arch=="ppc" or target_arch=="s390")', {
         'target_conditions': [
