--- chromium-72.0.3626.7.orig/third_party/blink/renderer/platform/text/character_property_data.h	2018-12-12 14:38:52.691743205 +0100
+++ chromium-72.0.3626.7/third_party/blink/renderer/platform/text/character_property_data.h	2018-12-12 14:49:27.320195000 +0100
@@ -247,7 +247,7 @@ static const UChar32 kIsHangulRanges[] =
     0xFFA0, 0xFFDC,
 };
 
-static const UChar32 kIsHangulArray[] = {};
+static const UChar32 kIsHangulArray[] = {0xFFDC,};
 
 #if !defined(USING_SYSTEM_ICU)
 // Freezed trie tree, see character_property_data_generator.cc.
