diff -urN chromium-60.0.3095.5.orig/third_party/WebKit/Source/platform/graphics/gpu/SharedGpuContext.h chromium-60.0.3095.5/third_party/WebKit/Source/platform/graphics/gpu/SharedGpuContext.h
--- chromium-60.0.3095.5.orig/third_party/WebKit/Source/platform/graphics/gpu/SharedGpuContext.h	2017-05-15 13:30:26.684754225 +0300
+++ chromium-60.0.3095.5/third_party/WebKit/Source/platform/graphics/gpu/SharedGpuContext.h	2017-05-15 13:31:22.272420471 +0300
@@ -8,6 +8,7 @@
 #include "platform/PlatformExport.h"
 #include "platform/wtf/ThreadSpecific.h"
 
+#include <functional>
 #include <memory>
 
 namespace gpu {
diff -urN chromium-60.0.3095.5.orig/v8/src/objects/hash-table.h chromium-60.0.3095.5/v8/src/objects/hash-table.h
--- chromium-60.0.3095.5.orig/v8/src/objects/hash-table.h	2017-05-15 13:30:23.953721493 +0300
+++ chromium-60.0.3095.5/v8/src/objects/hash-table.h	2017-05-15 13:32:01.427892344 +0300
@@ -138,22 +138,10 @@
  public:
   typedef Shape ShapeT;
 
-  // Wrapper methods
-  inline uint32_t Hash(Key key) {
-    if (Shape::UsesSeed) {
-      return Shape::SeededHash(key, GetHeap()->HashSeed());
-    } else {
-      return Shape::Hash(key);
-    }
-  }
-
-  inline uint32_t HashForObject(Key key, Object* object) {
-    if (Shape::UsesSeed) {
-      return Shape::SeededHashForObject(key, GetHeap()->HashSeed(), object);
-    } else {
-      return Shape::HashForObject(key, object);
-    }
-  }
+  // Wrapper methods.  Defined in src/objects-inl.h
+  // to break a cycle with src/heap/heap.h.
+  inline uint32_t Hash(Key key);
+  inline uint32_t HashForObject(Key key, Object* object);
 
   // Returns a new HashTable object.
   MUST_USE_RESULT static Handle<Derived> New(
diff -urN chromium-60.0.3095.5.orig/v8/src/objects-body-descriptors.h chromium-60.0.3095.5/v8/src/objects-body-descriptors.h
--- chromium-60.0.3095.5.orig/v8/src/objects-body-descriptors.h	2017-05-15 13:30:23.929721205 +0300
+++ chromium-60.0.3095.5/v8/src/objects-body-descriptors.h	2017-05-15 13:34:02.763404691 +0300
@@ -99,7 +99,7 @@
 
   template <typename StaticVisitor>
   static inline void IterateBody(HeapObject* obj, int object_size) {
-    IterateBody(obj);
+    IterateBody<StaticVisitor>(obj);
   }
 
   static inline int SizeOf(Map* map, HeapObject* object) { return kSize; }
diff -urN chromium-60.0.3095.5.orig/v8/src/objects-inl.h chromium-60.0.3095.5/v8/src/objects-inl.h
--- chromium-60.0.3095.5.orig/v8/src/objects-inl.h	2017-05-15 13:30:23.932721241 +0300
+++ chromium-60.0.3095.5/v8/src/objects-inl.h	2017-05-15 13:34:43.184896852 +0300
@@ -47,6 +47,25 @@
 namespace v8 {
 namespace internal {
 
+template <typename Derived, typename Shape, typename Key>
+uint32_t HashTable<Derived, Shape, Key>::Hash(Key key) {
+  if (Shape::UsesSeed) {
+    return Shape::SeededHash(key, GetHeap()->HashSeed());
+  } else {
+    return Shape::Hash(key);
+  }
+}
+
+template <typename Derived, typename Shape, typename Key>
+uint32_t HashTable<Derived, Shape, Key>::HashForObject(Key key,
+                                                       Object* object) {
+  if (Shape::UsesSeed) {
+    return Shape::SeededHashForObject(key, GetHeap()->HashSeed(), object);
+  } else {
+    return Shape::HashForObject(key, object);
+  }
+}
+
 PropertyDetails::PropertyDetails(Smi* smi) {
   value_ = smi->value();
 }
