diff -up chromium-61.0.3163.79/content/renderer/service_worker/web_service_worker_installed_scripts_manager_impl.cc.dde5e35 chromium-61.0.3163.79/content/renderer/service_worker/web_service_worker_installed_scripts_manager_impl.cc
diff -up chromium-61.0.3163.79/services/resource_coordinator/resource_coordinator_service.cc.dde5e35 chromium-61.0.3163.79/services/resource_coordinator/resource_coordinator_service.cc
--- chromium-61.0.3163.79/services/resource_coordinator/resource_coordinator_service.cc.dde5e35	2017-09-14 10:20:59.750146036 -0400
+++ chromium-61.0.3163.79/services/resource_coordinator/resource_coordinator_service.cc	2017-09-14 10:21:35.498261516 -0400
@@ -17,7 +17,8 @@ std::unique_ptr<service_manager::Service
   auto resource_coordinator_service =
       base::MakeUnique<ResourceCoordinatorService>();
 
-  return resource_coordinator_service;
+  return std::unique_ptr<service_manager::Service>(
+      resource_coordinator_service.release());
 }
 
 ResourceCoordinatorService::ResourceCoordinatorService()
