Index: chromium-72.0.3626.7/ui/gfx/linux/client_native_pixmap_dmabuf.cc
===================================================================
--- chromium-72.0.3626.7.orig/ui/gfx/linux/client_native_pixmap_dmabuf.cc
+++ chromium-72.0.3626.7/ui/gfx/linux/client_native_pixmap_dmabuf.cc
@@ -18,9 +18,6 @@
 #include "base/trace_event/trace_event.h"
 #include "build/build_config.h"
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4, 11, 0)
-#include <linux/dma-buf.h>
-#else
 #include <linux/types.h>
 
 struct dma_buf_sync {
@@ -35,7 +32,6 @@ struct dma_buf_sync {
 
 #define DMA_BUF_BASE 'b'
 #define DMA_BUF_IOCTL_SYNC _IOW(DMA_BUF_BASE, 0, struct dma_buf_sync)
-#endif
 
 namespace gfx {
 
