diff -up chromium-61.0.3163.100/chrome/browser/ui/webui/settings/on_startup_handler.cc.another-rvalue-fix chromium-61.0.3163.100/chrome/browser/ui/webui/settings/on_startup_handler.cc
--- chromium-61.0.3163.100/chrome/browser/ui/webui/settings/on_startup_handler.cc.another-rvalue-fix	2017-09-22 17:50:14.247727636 -0400
+++ chromium-61.0.3163.100/chrome/browser/ui/webui/settings/on_startup_handler.cc	2017-09-22 17:50:25.626492965 -0400
@@ -77,7 +77,7 @@ std::unique_ptr<base::Value> OnStartupHa
                    !extensions::ExtensionSystem::Get(profile_)
                         ->management_policy()
                         ->MustRemainEnabled(ntp_extension, nullptr));
-  return dict;
+  return std::move(dict);
 }

 void OnStartupHandler::HandleGetNtpExtension(const base::ListValue* args) {
