--- chromium-60.0.3112.78/tools/gn/BUILD.gn.lastcommitfix	2017-07-26 03:05:19.000000000 +0800
+++ chromium-60.0.3112.78/tools/gn/BUILD.gn	2017-07-26 16:22:57.418601999 +0800
@@ -268,7 +268,6 @@
 
   deps = [
     ":gn_lib",
-    ":last_commit_position",
     "//base",
     "//build/config:exe_and_shlib_deps",
     "//build/win:default_exe_manifest",
--- chromium-60.0.3112.78/tools/gn/gn_main.cc.lastcommitfix	2017-07-26 03:05:19.000000000 +0800
+++ chromium-60.0.3112.78/tools/gn/gn_main.cc	2017-07-26 16:22:57.432601941 +0800
@@ -12,13 +12,7 @@
 #include "tools/gn/standard_out.h"
 #include "tools/gn/switches.h"
 
-// Only the GN-generated build makes this header for now.
-// TODO(brettw) consider adding this if we need it in GYP.
-#if defined(GN_BUILD)
-#include "tools/gn/last_commit_position.h"
-#else
 #define LAST_COMMIT_POSITION "UNKNOWN"
-#endif
 
 namespace {
 
