diff -up chromium-65.0.3325.146/cc/paint/raw_memory_transfer_cache_entry.cc.memcpyfix chromium-65.0.3325.146/cc/paint/raw_memory_transfer_cache_entry.cc
--- chromium-65.0.3325.146/cc/paint/raw_memory_transfer_cache_entry.cc.memcpyfix	2018-03-07 11:04:14.690379817 -0500
+++ chromium-65.0.3325.146/cc/paint/raw_memory_transfer_cache_entry.cc	2018-03-07 11:06:04.339878069 -0500
@@ -3,7 +3,7 @@
 // found in the LICENSE file.
 
 #include "cc/paint/raw_memory_transfer_cache_entry.h"
-
+#include <memory.h>
 #include <string.h>
 
 namespace cc {
