diff -urN chromium-71.0.3578.30.orig/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py chromium-71.0.3578.30/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py
--- chromium-71.0.3578.30.orig/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py	2018-11-08 14:32:09.783181849 +0300
+++ chromium-71.0.3578.30/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py	2018-11-08 14:32:47.160705794 +0300
@@ -58,7 +58,7 @@
 
   with tempfile.NamedTemporaryFile() as _:
     args = [
-        'python',
+        'python2',
         rjsmin_path
     ]
     p = subprocess.Popen(args,
@@ -177,7 +177,7 @@
       os.path.join(py_vulcanize_path, 'third_party', 'rcssmin', 'rcssmin.py'))
 
   with tempfile.NamedTemporaryFile() as _:
-    rcssmin_args = ['python', rcssmin_path]
+    rcssmin_args = ['python2', rcssmin_path]
     p = subprocess.Popen(rcssmin_args,
                          stdin=subprocess.PIPE,
                          stdout=subprocess.PIPE,
diff -urN chromium-71.0.3578.30.orig/tools/gn/bootstrap/bootstrap.py chromium-71.0.3578.30/tools/gn/bootstrap/bootstrap.py
--- chromium-71.0.3578.30.orig/tools/gn/bootstrap/bootstrap.py	2018-11-08 14:33:17.817140200 +0300
+++ chromium-71.0.3578.30/tools/gn/bootstrap/bootstrap.py	2018-11-08 14:33:21.274189185 +0300
@@ -93,7 +93,7 @@
     if not options.debug:
       gn_gen_args += ' is_debug=false'
     subprocess.check_call([
-        gn_path, 'gen', out_dir,
+        gn_path, 'gen', '--script-executable=/usr/bin/python2', out_dir,
         '--args=%s' % gn_gen_args, "--root=" + SRC_ROOT
     ])
 
