diff -up chromium-69.0.3497.81/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py.py2 chromium-69.0.3497.81/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py
--- chromium-69.0.3497.81/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py.py2	2018-09-06 10:45:17.919774145 -0400
+++ chromium-69.0.3497.81/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py	2018-09-06 10:45:17.914774254 -0400
@@ -58,7 +58,7 @@ def _MinifyJS(input_js):
 
   with tempfile.NamedTemporaryFile() as _:
     args = [
-        'python',
+        'python2',
         rjsmin_path
     ]
     p = subprocess.Popen(args,
@@ -177,7 +177,7 @@ def _MinifyCSS(css_text):
       os.path.join(py_vulcanize_path, 'third_party', 'rcssmin', 'rcssmin.py'))
 
   with tempfile.NamedTemporaryFile() as _:
-    rcssmin_args = ['python', rcssmin_path]
+    rcssmin_args = ['python2', rcssmin_path]
     p = subprocess.Popen(rcssmin_args,
                          stdin=subprocess.PIPE,
                          stdout=subprocess.PIPE,
diff -up chromium-69.0.3497.81/tools/gn/bootstrap/bootstrap.py.py2 chromium-69.0.3497.81/tools/gn/bootstrap/bootstrap.py
--- chromium-69.0.3497.81/tools/gn/bootstrap/bootstrap.py.py2	2018-09-06 10:44:42.225550561 -0400
+++ chromium-69.0.3497.81/tools/gn/bootstrap/bootstrap.py	2018-09-06 13:06:24.251636231 -0400
@@ -87,7 +87,7 @@ def main(argv):
   if not options.debug:
     gn_gen_args += ' is_debug=false'
   subprocess.check_call([
-      gn_path, 'gen', out_dir,
+      gn_path, 'gen', out_dir, ' --script-executable=/usr/bin/python2',
       '--args=%s' % gn_gen_args, "--root=" + SRC_ROOT
   ])
 
