From 8f49c61779610638bf4313000c0e9f467523730e Mon Sep 17 00:00:00 2001
From: Akarshan Biswas <hellbanger@ga-lmt-s2.biswasab.org>
Date: Sat, 20 Oct 2018 09:53:26 +0530
Subject: [PATCH] Silence some gcc warnings on linux

---
 build/config/compiler/BUILD.gn | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index d1bb5597f..05aae1063 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -239,6 +239,14 @@ config("compiler") {
     ":compiler_cpu_abi",
     ":compiler_codegen",
   ]
+  # Avoid warning flood on linux with gcc
+   if (is_linux && !is_clang) {
+     cflags += [ "-Wno-pragmas" ]
+     cflags += [ "-Wno-attributes" ]
+     cflags += [ "-Wno-ignored-attributes" ]
+     cflags += [ "-Wno-packed-not-aligned" ]
+     cflags_cc += [ "-Wno-class-memaccess" ]
+   }
 
   # In general, Windows is totally different, but all the other builds share
   # some common GCC configuration.
-- 
2.17.2

