diff --git a/chrome/services/file_util/public/cpp/BUILD.gn b/chrome/services/file_util/public/cpp/BUILD.gn
index 88c2008..f55e4fd 100644
--- a/chrome/services/file_util/public/cpp/BUILD.gn
+++ b/chrome/services/file_util/public/cpp/BUILD.gn
@@ -26,11 +26,6 @@ source_set("cpp") {
       ]
     }
 
-    #TODO(crbug/750327): This dependency is here temporarily.
-    deps = [
-      "//third_party/unrar:unrar",
-    ]
-
     public_deps += [ "//chrome/common/safe_browsing" ]
   }
 }
